src/basic/openssl-util.h

```diff
@@ -21,12 +22,22 @@ int hmac(
         uint8_t *md,
         unsigned int *md_len);
 
+int openssl_hash(
```

Comment:
hmm, if we prefix this function with `openssl_`, maybe do the same for `hmac()` above?